glue_jupyter/bqplot/histogram/__init__.py

```diff
@@ -1,2 +1,6 @@
 from .layer_artist import *  # noqa
 from .viewer import *  # noqa
+
+
+def setup():
+    pass
```

Comment:
Ideally the ``setup`` function should actually be adding the viewer to the registry (rather than using the decorator) - @CyclingNinja is working on improvements to ``load_plugins`` that means that users will be able to opt-in to specific plugins, so we need the viewer to not be added if the entry point is not called.